_SQLServer/TableLoadHeroes.sql

```diff
@@ -0,0 +1,478 @@
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_One_Shot_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_One_Shot_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/One_Shot.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'With the crack of her rifle and the smell of gunpowder, One Shot brings her signature ranged warfare into Super Dungeon Explore with deadly effect.  Terrified monsters flee in panic, but they can''t run fast enough.  One Shot has them in her sights!');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Daystar 
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Focused Shot
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Fritz
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 6, 3, '2R', '3B', '2B', '2B', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,0,1);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,2);--! To look at later
+
+
+--Princess Malya
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Princess Malya', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Princess_Malya_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Princess_Malya_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Princess_Malya.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'Even a racing super-star has time to sit down to a game of Super Dungeon Explore.  Princess Malya brings her love of speed into the Last Galaxy''s favorite online game, zipping through the dungeon at a breakneck pace.  Malya slays monsters with glee, but is always willing to lend fellow adventurers a hand with the help of some creation magic.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Energy Blast
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Purification
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Mr. Tomn
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 8, 3, '1B1R', '2B', '2B', '2R', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,0,1);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,4);--! To look at later
```

Comment:
This character should have a 1st defense assignment for armor (0,4)

Reply:
This should be a (0,2), (0,4) already there - per our conversation on 2/27/21.